libraries/Microsoft.Bot.Builder.LanguageGeneration/StaticChecker.cs

```diff
@@ -386,9 +387,14 @@ private List<Diagnostic> CheckExpression(string exp, ParserRuleContext context,
             ParserRuleContext context = null)
         {
             var lineOffset = this.currentTemplate != null ? this.currentTemplate.SourceRange.Range.Start.Line : 0;
-            message = this.currentTemplate != null ? $"[{this.currentTemplate.Name}]" + message : message;
+            var templateNameInfo = string.Empty;
+            if (this.currentTemplate != null && this.currentTemplate.Name != "__temp__")
```

Comment:
>"__temp__" [](start = 77, length = 10)

Might want to make this a constant and then used everywhere--makes typos less likely.